components/filter/script.js

```
@@ -0,0 +1,410 @@
+
+async function updateDataBasedOnFilters({
+  renderFunction,
+  filterStatus,
+  page,
+  parentContainer,
+  otherFilters,
+}) {
+  if (page === 'requests') {
+    parentContainer.innerHTML = '';
+    const params = new URLSearchParams(window.location.search);
+    let currentReqType = params.get('type')?.toUpperCase() ?? 'OOO';
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'currentReqType' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)